### ISSUE TITLE
Error: element located, but timed out after 30 seconds; Regarding: text_field and input

### ISSUE THREAD

**Issue by mcshakes** [watir/watir#703]
## Meta -
Watir Version: 6.7.3
Selenium Version: 
Browser Version: Chrome 63.0.3239.132 (Official Build) (64-bit)
Browser Driver Version: Chromedrive 2.27
OS Version: OSX 10.10.5

## Expected Behavior -
Running this code should set text to text_field:

`@browser.input(name: "add_comment_text").send_keys "TEST"`

`@browser.text_field(name: "add_comment_text").set "TEST"`

## Actual Behavior -

Instead, I get the error:
`Watir::Exception::UnknownObjectException: element located, but timed out after 30 seconds, waiting for #<Watir::Input: located: true; {:name=>"add_comment_text", :tag_name=>"input"}> to be present
from /Users/EdwardMcCarthy/.rvm/gems/ruby-2.4.0/gems/watir-6.7.3/lib/watir/elements/element.rb:692:in `raise_present'`

## Steps to reproduce -

So, I'm trying to automate behavior where a user would enter text in a textfield below:

![error_img](https://user-images.githubusercontent.com/8164007/35004265-9ada444c-faac-11e7-9189-7396f1e9520a.png)

I tried the following to pinpoint where I was going wrong:

`@browser.text_field(name: "add_comment_text").exists?` ==> true

`@browser.text_field(name: "add_comment_text").visible?` ==> false

`@browser.text_field(name: "add_comment_text").visible?` ==> false

- Clearly I can see it the text_field.

I found something similar in this issue: https://github.com/watir/watir/issues/566, but the workaround using `Watir.relaxed_locate = false`  does not work for me. Let me know if I can get a lot more info, especially the actual HTML I am trying to navigate.

**Comment by titusfortner** [comment]
Firstly, try updating your chromedriver. Chrome 63 requires using chromedriver > 2.33
I recommend using [`webdrivers` gem](https://github.com/titusfortner/webdrivers) to keep the driver updated automatically.

If updating the driver doesn't work, can you provide the full stack trace? There should be more information in the error message.

**Comment by mcshakes** [comment]
After just updating to Watir 6.10.2, I reran the following scripts:

1) `@browser.text_field(name: "add_comment_text").set "TEST COMMENTS!!"`

and got:

`Watir::Exception::ObjectDisabledException: element present and enabled, but timed out after 30 seconds, waiting for #<Watir::TextField: located: true; {:name=>"add_comment_text", :tag_name=>"input"}> to not be disabled
from /Users/EdwardMcCarthy/.rvm/gems/ruby-2.4.0/gems/watir-6.10.2/lib/watir/elements/element.rb:619:in `raise_disabled'`

As checks, I ran:

`@browser.text_field(name: "add_comment_text").enabled?` ==> true

`@browser.text_field(name: "add_comment_text").present?` ==> false

2) `@browser.input(name: "add_comment_text").send_keys "HEY"`

response:

`Watir::Exception::UnknownObjectException: element located, but timed out after 30 seconds, waiting for #<Watir::Input: located: true; {:name=>"add_comment_text", :tag_name=>"input"}> to be present
from /Users/EdwardMcCarthy/.rvm/gems/ruby-2.4.0/gems/watir-6.10.2/lib/watir/elements/element.rb:623:in `raise_present'`

I will try updating the chromedriver next; just wanted to be thorough and paint a bigger/clearer picture

**Comment by jkotests** [comment]
Are you sure that users directly interact with the `<input name="add_comment_text">` element?

In the screenshot, the developer toolbar has a `<div contenteditable="true" role="textbox">`. Are users perhaps interacting with this `div` tag that is designed to behave like a text field?

**Comment by titusfortner** [comment]
> element present and enabled, but timed out after 30 seconds, waiting for <> to not be disabled

Sigh, yeah, that's a confused app. Even if you get your stuff working, this error message shouldn't ever happen. Can you run this with `Watir.logger.level = :info`

We'll dig into it.

**Comment by jkotests** [comment]
@titusfortner, what do you mean that error message shouldn't happen? Are you referring to the message saying the element is "enabled" but waiting for it to "not be disabled"? That just seems to be a typo in the Watir code:

~~~~~~~~
def raise_disabled
  message = "element present and enabled, but timed out after #{Watir.default_timeout} seconds, waiting for #{inspect} to not be disabled"
  raise ObjectDisabledException, message
end
~~~~~~~~

**Comment by mcshakes** [comment]
@jkotests I will poke around with other divs (there are quite a few which dynamically changing id/classes). Entirely possible I'm wrong with my selector

An update: 
- updated to chromedriver 2.35, and still ran into the same errors with it.
- Set `Watir.logger.level = :info` and re-ran `@browser.text_field(name: "add_comment_text").set "HEEY"`

Got a long list of errors detailing watir completing assertions and ending with the same previous error. Want me to paste the entire stack trace here?

**Comment by titusfortner** [comment]
link to a http://gist.github.com

**Comment by mcshakes** [comment]
Here you go:

https://gist.github.com/mcshakes/b147249c02c2b20486b0b5c7f6dc65c7

Also, @jkotests you were correct, I was trying to write to the wrong HTML inputs. Instead, I should have been targeting:

```
<div aria-autocomplete="list" aria-controls="js_2dj" aria-describedby="placeholder-c2oua" aria-multiline="true" class="notranslate _5rpu" contenteditable="true" data-testid="ufi_comment_composer" role="textbox" spellcheck="true" style="outline: none; user-select: text; white-space: pre-wrap; word-wrap: break-word;">
    <div data-contents="true">
        <div class="" data-block="true" data-editor="c2oua" data-offset-key="7ltls-0-0">
            <div data-offset-key="7ltls-0-0" class="_1mf _1mj">
                <span data-offset-key="7ltls-0-0">
                    <br data-text="true">
                </span>
            </div>
        </div>
    </div>
</div>
```

**Comment by titusfortner** [comment]
And, I finally figured out why the error message was dumb.
Fixed: ec7bb9b01c3a9cb74b992c7ef18956fee9f1e6ad

Thanks
